tests/api/test_security.py

```diff
@@ -1,45 +1,44 @@
 import requests
-import unittest
 import os
 
 
-class Testsecurity(object):
+class TestPOST(object):
     def setup(self):
         # app['TESTING'] = True
         # self.app = app.test_client()
         pass
 
-    def test_main_page(self, flask_api_url):
-        response = requests.get(flask_api_url + '/')
+    def test_main_page(self, session, flask_api_url):
+        response = session.get(flask_api_url + '/')
         assert response.status_code == 200
 
-    def test_login_fake_user(self, flask_api_url):
-        response = requests.post(flask_api_url + '/api/users/login',
+    def test_login_fake_user(self,session, flask_api_url):
+        response = session.post(flask_api_url + '/api/users/login',
                                  data=dict(email='user1', password='password'))
 
         assert response.reason == 'FORBIDDEN'
         assert response.status_code == 403
 
-    def test_login_real_user(self, flask_api_url):
-        response = requests.post(flask_api_url + '/api/users/login',
+    def test_login_real_user(self,session, flask_api_url):
+        response = session.post(flask_api_url + '/api/users/login',
```

Comment:
session already logs you in in the fixture. Please actually do session = requests.Session() in the method if you want to test login/out